src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="card container__card" data-qa="card">
+        <a href="#!" class="card__link">
+          <img
+            src="images/imac.jpeg"
+            alt="Moyo-Product-imac/APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__img"
+          >
+        </a>
+
+
+        <a href="#!" class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+
+        <div class="card__product-code">Product code: 195434</div>
+        <a href="#!" class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__stars-review">Reviews: 5</span>
+        </a>
+
+
+        <div class="card__prices">
+          <div class="card__price">Price:</div>
+          <div class="card__value">$2,199</div>
+        </div>
```

Comment:
I'd rather do so
NB: `-` is used to conjugate long term, it does not affect initial BEM structure
```suggestion
        <div class="card__price">
          <div class="card__price-label">Price:</div>
          <div class="card__price-value">$2,199</div>
        </div>
```